examples/lock-app/nrfconnect/main/AppTask.cpp

```diff
@@ -212,6 +213,7 @@ CHIP_ERROR AppTask::Init()
     (void) initParams.InitializeStaticResourcesBeforeServerInit();
     initParams.testEventTriggerDelegate = &testEventTriggerDelegate;
     ReturnErrorOnFailure(chip::Server::GetInstance().Init(initParams));
+    AppFabricTableDelegate::Init();
```

Comment:
Maybe planned.. but we should do it for all the samples.

Reply:
Yes, of course, this draft points only to the Lock to be clearer for the review :)